scripts/asr_language_modeling/ngram_lm/eval_beamsearch_ngram_transducer.py

```diff
@@ -0,0 +1,264 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# This script would evaluate an N-gram language model trained with KenLM library (https://github.com/kpu/kenlm) in
+# fusion with beam search decoders on top of a trained ASR model. NeMo's beam search decoders are capable of using the
+# KenLM's N-gram models to find the best candidates. This script supports both character level and BPE level
+# encodings and models which is detected automatically from the type of the model.
+# You may train the LM model with 'scripts/ngram_lm/train_kenlm.py'.
+#
+# USAGE: python eval_beamsearch_ngram.py  --nemo_model_file=<path to the .nemo file of the model> \
+#                                         --input_manifest=<path to the evaluation JSON manifest file \
+#                                         --kenlm_model_file=<path to the binary KenLM model> \
+#                                         --beam_width=<list of the beam widths> \
+#                                         --beam_alpha=<list of the beam alphas> \
+#                                         --preds_output_folder=<optional folder to store the predictions> \
+#                                         --decoding_mode=maes
+#                                         ...
+#
+# You may find more info on how to use this script at:
```

Comment:
Let's add it to the main doc page that we now support KenLM for both CTC and Transducer models and add the link to this script to the main page at the following address.
Please also add it to the main README page of nemo doc if missing.

Reply:
I can do it in another PR (merging `eval_beamsearch_ngram_transducer.py` with `eval_beamsearch_ngram.py`) which I mentioned above.